packages/upward-js/lib/resolvers/__tests__/TemplateResolver.test.js

```diff
@@ -42,6 +42,7 @@ test('throws if template engine is unsupported', async () => {
         ),
         context: {
             get: jest.fn(() => {
+                // eslint-disable-next-line no-unused-labels
```

Comment:
Look more closely! This is a bug that's been caught by the linter.

To implicitly return an object from an arrow function, we have to wrap the `{}` in `()`. Since that didn't happen here, what should have been an object property was interpreted as a label, which has been flagged by the linter as unused.

```es6
get: jest.fn(() => ({
    ENV_VAR: 'ENV_VALUE'
}))
```

Reply:
I wonder if this test is just busted then... I'll fix.